examples/resources/kentik-synthetic_test-bulk-create/agents_filtered_by_asn/resource.tf

```diff
@@ -23,7 +23,6 @@ resource "kentik-synthetics_test" "agents-filtered-by-asn-test" {
       "ping",
       "traceroute"
     ])
-    health_settings {}
```

Comment:
Do we default `healthSettings` to an empty dict in creation requests (in the provider or SDK)? Kentik API (unfortunately) requires the empty `healthSettings` dict to be present in the JSON payload. Without it creation request fails with 400 with are very long error message:
```
{
  "code": 3,
  "message": "internal validation (config.healthSettings.latencyCritical: Cannot read property 'latencyCritical' of undefined, Cannot read property 'latencyCritical' of undefined; config.healthSettings.latencyWarning: Cannot read property 'latencyWarning' of undefined, Cannot read property 'latencyWarning' of undefined; config.healthSettings.packetLossCritical: Cannot read property 'packetLossCritical' of undefined, Cannot read property 'packetLossCritical' of undefined; config.healthSettings.packetLossWarning: Cannot read property 'packetLossWarning' of undefined, Cannot read property 'packetLossWarning' of undefined; config.healthSettings.jitterCritical: Cannot read property 'jitterCritical' of undefined, Cannot read property 'jitterCritical' of undefined; config.healthSettings.jitterWarning: Cannot read property 'jitterWarning' of undefined, Cannot read property 'jitterWarning' of undefined; config.healthSettings.httpLatencyCritical: Cannot read property 'httpLatencyCritical' of undefined, Cannot read property 'httpLatencyCritical' of undefined; config.healthSettings.httpLatencyWarning: Cannot read property 'httpLatencyWarning' of undefined, Cannot read property 'httpLatencyWarning' of undefined; config.healthSettings.httpValidCodes: Cannot read property 'httpValidCodes' of undefined; config.healthSettings.dnsValidCodes: Cannot read property 'dnsValidCodes' of undefined) (errxid c4e9f44p64h13xzsxjv0)",
  "details": []
}
```

Reply:
Yes, if not present, empty health settings are provided:
```

      "health_settings" = tolist([
        {
          "dns_valid_codes" = tolist([])
          "http_latency_critical" = 0
          "http_latency_warning" = 0
          "http_valid_codes" = tolist([])
          "jitter_critical" = 0
          "jitter_warning" = 0
          "latency_critical" = 0
          "latency_warning" = 0
          "packet_loss_critical" = 0
          "packet_loss_warning" = 0
        },
      ])
```

Reply:
The above should work. When I create test passing empty dictionary (`{}`) in `healthSettings` the test is created with `healthSettings` matching the above. The JSON looks like this:
```json
      "healthSettings": {
        "latencyCritical": 0,
        "latencyWarning": 0,
        "packetLossCritical": 0,
        "packetLossWarning": 0,
        "jitterCritical": 0,
        "jitterWarning": 0,
        "httpLatencyCritical": 0,
        "httpLatencyWarning": 0,
        "httpValidCodes": [],
        "dnsValidCodes": []
      }
```